src/transpiler.js

```diff
@@ -51,18 +51,13 @@ function getAddress({ addresses = [] }) {
 }
 
 function getBirthday({ birthdays = [] }) {
-  return birthdays
-    .filter(
-      birthday => birthday && birthday.metadata && birthday.metadata.primary
-    )
-    .map(
-      birthday =>
-        birthday &&
-        birthday.date &&
-        /\d\d\d\d-\d\d-\d\d/.exec(
-          new Date(Object.values(birthday.date).join('-')).toISOString()
-        )[0]
-    )[0]
+  const birthday =
+    birthdays.find(b => b.metadata && b.metadata.primary) || birthdays[0]
+  if (birthday && birthday.date) {
+    const { year, month, day } = birthday.date
+    const isoDate = new Date(year, month - 1, day, 10, 0).toISOString()
```

Comment:
Why 10 hours?^^

Reply:
This is the fix in fact. I'm still searching how to do this in another way in js. Maybe with `Date.UTC` : https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Date/UTC

Reply:
just force pushed. It's much more explicit with `Date.UTC`

Reply:
Oook yes, now I understand! LGTM like this!